sqlite_query.c

```diff
@@ -268,7 +272,11 @@ sqlite_convert_to_pg(Form_pg_attribute att, sqlite3_value * val, AttInMetadata *
 					{
 						if (value_byte_size_blob_or_utf8)
 						{
```

Comment:
I think the SQLITE3_TEXT case should be placed above the default case.
Maybe 
```
case SQLITE_FLOAT:
      {}
case SQLITE3_TEXT:
      {}
case SQLITE_INTEGER:
case SQLITE_BLOB:
default:
     {sqlite_value_to_pg_error()}
```

Reply:
Changed to proposed order. For `FLOAT*OID`s there is no reason for my previous order. Please check, @MinhLA1410.

Reply:
Follow the old code, SQLITE_INTEGER and SQLITE_BLOB will be process same as default (raise error message)
But with your new change, SQLITE_INTEGER and SQLITE_BLOB will be process same as SQLITE3_TEXT.
Is it correct?
If it's correct,  is there any testcases covering this case (SQLITE_INTEGER and SQLITE_BLOB in FLOAT*OID)?

Reply:
> But with your new change, SQLITE_INTEGER and SQLITE_BLOB will be process same as SQLITE3_TEXT.
> Is it correct?

No. This will be fixed in some minutes. I am sorry, this was mechanical edit with successfully  tests.

Reply:
This should be fixed in https://github.com/pgspider/sqlite_fdw/commit/9cc95e39ecbb4e6968e162d004f126c4b42cef04.

Reply:
Your modification in https://github.com/pgspider/sqlite_fdw/commit/9cc95e39ecbb4e6968e162d004f126c4b42cef04 make to src code back to original. Please change order of FLOAT*OID 
from: SQLITE_FLOAT-> SQLITE_INTEGER -> SQLITE_BLOB -> SQLITE3_TEXT -> default
to    : SQLITE_FLOAT-> SQLITE3_TEXT -> SQLITE_INTEGER -> SQLITE_BLOB -> default
It will be remove the duplicate code 
```
case SQLITE_FLOAT:
      {}
case SQLITE_INTEGER:
case SQLITE_BLOB:
      {
             sqlite_value_to_pg_error()    <--- duplicate code, please move SQLITE_INTEGER, SQLITE_BLOB to under SQLITE3_TEXT
             break;
      }
case SQLITE3_TEXT:
      {}
default:
     {
             sqlite_value_to_pg_error()
             break;
      }
```

Reply:
Thank you. It's OK.